### ISSUE TITLE
Links should be displayed in Reminders

### ISSUE THREAD

**Issue by ilMagnifico** [pakerwreah/Calendr#199]
Reminders don’t display links (if they have one). Maybe they could show the link when I click on them.

<img width="181" alt="Screenshot 2024-06-27 alle 15 06 44" src="https://github.com/pakerwreah/Calendr/assets/16544166/fbc46eca-bc7b-4d56-9ade-f6fdfce158b5">
<img width="375" alt="Screenshot 2024-06-27 alle 15 07 14" src="https://github.com/pakerwreah/Calendr/assets/16544166/a6c8c158-d155-408f-b84a-e279a20b6cb5">

**Comment by pakerwreah** [comment]
🤔 yeah, should.

**Comment by pakerwreah** [comment]
ok, that's actually a bug in Apple's api. I even left a couple of comments in the code a long time ago:
<img width=450 src='https://github.com/pakerwreah/Calendr/assets/803954/ddb2a63c-8580-44c8-9c4b-61e34bb47f57' />

If you place your link in the reminder's `Notes` section, it works, because that's one of the places I check.

Perhaps with the new events+reminders integration in the next macOS, they end up fixing it.

**Comment by ilMagnifico** [comment]
> ok, that's actually a bug in Apple's api. I even left a couple of comments in the code a long time ago: <img alt="" width="450" src="https://private-user-images.githubusercontent.com/803954/344403797-ddb2a63c-8580-44c8-9c4b-61e34bb47f57.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk2ODI2NzIsIm5iZiI6MTcxOTY4MjM3MiwicGF0aCI6Ii84MDM5NTQvMzQ0NDAzNzk3LWRkYjJhNjNjLTg1ODAtNDRjOC05YzRiLTYxZTM0YmI0N2Y1Ny5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYyOVQxNzMyNTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zMjQ3MzdkZmU3OTQ1OTUyMDlmMzJjYzRhNmY5YjYxNmM0ZTU1M2MzMGIwZDY3MjIxZDljNDRiZjVkNmQ2NDk3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.pLrISlvp0eSrYnNGHdWGk2i1FYkMdxt0LNLMNz_12TI">
> 
> If you place your link in the reminder's `Notes` section, it works, because that's one of the places I check.
> 
> Perhaps with the new events+reminders integration in the next macOS, they end up fixing it.

Have you [reported](https://developer.apple.com/bug-reporting/) this?

**Comment by pakerwreah** [comment]
This bug is widely known, I found a bunch of issues open with no solution. Let's wait for the new macOS release and see if anything changes.

**Comment by ilMagnifico** [comment]
Does this bug still exists?

**Comment by pakerwreah** [comment]
yes, if Apple fixes it, it should work automatically, because I'm trying to get the urls.

**Comment by ilMagnifico** [comment]
> yes, if Apple fixes it, it should work automatically, because I'm trying to get the urls.

So It's not currently work.

**Comment by ilMagnifico** [comment]
Hoping the new OS will fix it.
